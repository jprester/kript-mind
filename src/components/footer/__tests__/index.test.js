import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../index';

const wrapper = shallow((<Footer />));

describe('index', () => {
  it('should render Footer correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
