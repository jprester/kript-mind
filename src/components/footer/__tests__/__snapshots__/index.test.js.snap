// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should render Footer correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="main-container"
>
        <div
                className="footer-text-left"
        >
                v
                0.1.0
        </div>
        <div
                className="footer-text-right"
        >
                KriptMind, Janko Prester 2018
        </div>
</div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="footer-text-left"
>
            v
            0.1.0
</div>,
          <div
            className="footer-text-right"
>
            KriptMind, Janko Prester 2018
</div>,
        ],
        "className": "main-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "v",
              "0.1.0",
            ],
            "className": "footer-text-left",
          },
          "ref": null,
          "rendered": Array [
            "v",
            "0.1.0",
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "KriptMind, Janko Prester 2018",
            "className": "footer-text-right",
          },
          "ref": null,
          "rendered": "KriptMind, Janko Prester 2018",
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="main-container"
>
          <div
                    className="footer-text-left"
          >
                    v
                    0.1.0
          </div>
          <div
                    className="footer-text-right"
          >
                    KriptMind, Janko Prester 2018
          </div>
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="footer-text-left"
>
              v
              0.1.0
</div>,
            <div
              className="footer-text-right"
>
              KriptMind, Janko Prester 2018
</div>,
          ],
          "className": "main-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "v",
                "0.1.0",
              ],
              "className": "footer-text-left",
            },
            "ref": null,
            "rendered": Array [
              "v",
              "0.1.0",
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "KriptMind, Janko Prester 2018",
              "className": "footer-text-right",
            },
            "ref": null,
            "rendered": "KriptMind, Janko Prester 2018",
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
